### PR TITLE
Add functionality to the sessions user link in admin/users

### LINKED ISSUE #1216: Add a route to which the admin gets redirected when sessions is clicked in admin/users
**Describe the bug**

image
When sessions is clicked nothing happens in the route admin/users
**To Reproduce**
1. Go to admin/users
2. Click on sessions in the user links column

**Expected behavior**

It should show sessions of that particular user.

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser: Chromium

### PR DESCRIPTION
**Pull request by ritikamotwani**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
When the session is clicked in the route admin/users the admin gets redirected nowhere.

#### Changes proposed in this pull request:

- A route is added so that the admin can view the sessions of all users.
- When sessions user link is clicked by the admin the admin gets redirected to that route.

Fixes #1216

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -120,6 +120,9 @@ router.map(function() {
     this.route('users', function() {
       this.route('list', { path: '/:users_status' });
       this.route('user', { path: '/u/:user_id' });
+      this.route('user-sessions', function() {
```
Comment (niranjan94) [inline app/router.js]: What is this `/user-sessions` route for ? 

Sessions for a user should be located at `/users/<id>/sessions/<session_status>`

Comment (ritikamotwani) [reply app/router.js]: @niranjan94 now I have created the route as /users/sessions/<user_id>/<session_status> as for creating it as /users/<id>/sessions/<session_status> I had to introduce some other path in between.

**Comment by ritikamotwani** [comment]
@niranjan94 now I have created the route as /users/sessions/<user_id>/<session_status> as for creating it as /users//sessions/<session_status> I had to introduce some other path in between.

File: app/router.js [hunk 2]
```diff
@@ -120,6 +120,9 @@ router.map(function() {
     this.route('users', function() {
       this.route('list', { path: '/:users_status' });
       this.route('user', { path: '/u/:user_id' });
```
Comment (niranjan94) [inline app/router.js]: @ritikamotwani use this route. This already accepts a `user_id`.

Comment (niranjan94) [inline app/router.js]: @ritikamotwani use this route. This already accepts a `user_id`. 

This is exactly what is needed. Change any other route as needed to make the structure as follows.

`/admin/users/:user_status` - List of users
`/admin/users/:user_id` - Viewing a specific user
`/admin/users/:user_id/sessions/:session_status` - Listing sessions that belong to a user. 

For the first two, checkout `events.view` and `events.list` for inspiration on how to go about it.

File: app/router.js [hunk 3]
```diff
@@ -119,7 +119,11 @@ router.map(function() {
     });
     this.route('users', function() {
       this.route('list', { path: '/:users_status' });
-      this.route('user', { path: '/u/:user_id' });
+      this.route('user', { path: '/u/:user_id' }, function() {
```
Comment (niranjan94) [inline app/router.js]: Like I said before, No `/u` ... it should just be `/:user-id`

**Comment by ritikamotwani** [comment]
@niranjan94 did the changes for the route.

File: app/router.js [hunk 4]
```diff
@@ -118,8 +118,12 @@ router.map(function() {
       this.route('list', { path: '/:sessions_state' });
     });
     this.route('users', function() {
+      this.route('user', { path: '/:user_id' }, function() {
```
Comment (niranjan94) [inline app/router.js]: call this `view` to match with conventions followed everywhere else

**Comment by ritikamotwani** [comment]
@niranjan94 renamed it to view.

**Comment by ritikamotwani** [comment]
@niranjan94 please review

**Pull request merged**